index.html

```diff
@@ -3,55 +3,82 @@
   <head>
     <meta charset="UTF-8">
     <title>gci15.fossasia.org by fossasia</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
-    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
+    <!--Correct the .css path once you have put both files in a folder-->
+    <link rel="stylesheet" type="text/css" href="C:\Users\emanf.ETHANSCOMPUTER\Desktop\FOSSASIA Website\stylesheet.css" media="screen">
```

Comment:
This would not work outside your computer. change it to just `stylesheet.css`  

`C:\Users\emanf.ETHANSCOMPUTER\Desktop\FOSSASIA Website\stylesheet.css` -> `stylesheet.css`